packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -443,6 +457,13 @@ export interface AddAutoScalingGroupCapacityOptions {
    * @default 300
    */
   readonly taskDrainTimeSeconds?: number;
+
+  /**
+   * The machine image for the ECS instances
+   *
+   * @default - No default machine image
```

Comment:
You want to add this to `AddCapacityOptions`, not to `AddAutoScalingGroupCapacityOptions`.

Also, the default mentioned here is not true, is it? The default is an ECS-optimized Amazon Linux 1 image.

Reply:
Ping on this comment

Reply:
OK, have pushed a new commit to move `machineImage` param to `AddCapacityOptions`